index.js

```diff
@@ -1,81 +1,120 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
-
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+ console.log("%cQuestion 1", "color: red") 
+const counter = 1
+console.log(counter) 
+// => 2
+ console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+ console.log("%cQuestion 2", "color: red")
 
-// console.log(name)
-// => "Raffy"
-// name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+
+//=> "Raffy"
+const name = "menachem" 
+console.log(name)
+//=> TypeError
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+ console.log("%cQuestion 3", "color: red") 
+
+ function drinkWater(current_thirst_level){
+  console.log ("Man I sure am thirsty")
+  current_thirst_level -= 1
+  console.log ("Ahh that hits the spot")
+  return current_thirst_level
+ }
```

Comment:
Great work transforming Ruby into JS. 

remember that the standard/norm for declaring variables in ruby is snake case and the standard in JS is camel case. 
To clean up this JS I would change the variable from `current_thirst_level ` to `currentThirstLevel`. 

NOTE: both work and do not affect the outcome of your function.